examples/rollup/package.json

```diff
@@ -20,7 +20,6 @@
   "scripts": {
     "build": "rollup -c",
     "server": "FLASK_APP=server.py python -m flask run --port 4242",
-    "local-install": "yalc add @stripe/connect-js@1.0.0 && yalc add @stripe/react-connect-js@1.0.0",
```

Comment:
How will we test the local changes with this sample?

Reply:
I think the goal of this sample is not to test local changes but to show users how to use this library.

Reply:
Looks like even StripeJS doesn't have that: https://github.com/stripe/stripe-js/blob/master/CONTRIBUTING.md. I am peaceful with these changes given that, it'd be great if we could document how to use yalc to test local changes in this example in the readme.md of the examples/rollup folder, or in the CONTRIBUTING.md. I can see us asking someone external "can you test ..." however if there are no instructions, that is hard.

Reply:
Added some instructions in readme!